core/imt/src/TPoolManager.cxx

```diff
@@ -1,4 +1,4 @@
-#include "ROOT/TPoolManager.hxx"
+#include "TPoolManager.hxx"
```

Comment:
Isn't this a breaking change for users?

Reply:
No. It's internal to ROOT and I'm the only one that has ever used it afaik